executable/mach_o.ksy

```diff
@@ -24,12 +24,13 @@ seq:
     repeat-expr: header.ncmds
 enums:
   magic_type:
+    # Note that for multiarch (a.k.a. fat) Mach-O files, which are the primary
+    # kind you find on macOS today, you should instead use mach_o_fat.ksy, which
+    # parses the fat header and embeds us to parse each arch.
```

Comment:
> and embeds **_us_** to parse each arch.

I wouldn't use "embeds *us*" here - depending on how you read it, it may not be immediately clear that "*us*" refers to "this `mach_o.ksy` specification".